trojan-quickstart.sh

```diff
@@ -67,6 +67,8 @@ Type=simple
 StandardError=journal
 ExecStart="$BINARYPATH" "$CONFIGPATH"
 ExecReload=/bin/kill -HUP \$MAINPID
+LimitNOFILE=51200
+LimitNPROC=51200
```

Comment:
Why do we need this?

Reply:
maybe not but no harm to set the limitation higher,is 't it?

Reply:
The maximum number of threads that trojan will open is 2. There is no point setting this. I suggest deleting it to avoid causing further confusion.
```suggestion
```